src/App.jsx

```diff
@@ -1,26 +1,26 @@
-import React, { useEffect } from "react";
-import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+// import React, { useEffect } from "react";
+// import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
 
-import Login from "./login/login.jsx";
-import HomePage from "./HomePage";
-import Register from "./login/register.jsx";
-import { AuthProvider } from "./contexts/authContext";
+// import Login from "./login/login.jsx";
+// import HomePage from "./HomePage";
+// import Register from "./login/register.jsx";
+// import { AuthProvider } from "./contexts/authContext";
 
-function App() {
-  return (
-    <AuthProvider>
```

Comment:
@Capn05 a reminder to add styling as you go so it doesn't add up and we can provide feedback overtime.